test/runtime/Policies.go

```diff
@@ -994,6 +990,74 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 		res.ExpectFail("unexpectedly able to ping %s", helpers.App2)
 
 	})
+
+	Context("TestsEgressToHost", func() {
+
+		hostDockerContainer := "hostDockerContainer"
+
+		BeforeEach(func() {
+			By("Starting httpd server using host networking")
+			res := vm.ContainerCreate(hostDockerContainer, helpers.HttpdImage, helpers.HostDockerNetwork, "-l id.hostDockerContainer")
+			res.ExpectSuccess("unable to start Docker container with host networking")
+		})
+
+		AfterEach(func() {
+			vm.ContainerRm(hostDockerContainer)
+		})
+
+		It("Tests Egress To Host", func() {
+
+			app1Label := "id.app1"
+			hostIP := "10.0.2.15"
+
+			By(fmt.Sprintf("Pinging %s from %s before importing policy (should work)", api.EntityHost, helpers.App1))
+			failedPing := vm.ContainerExec(helpers.App1, helpers.Ping(hostIP))
+			failedPing.ExpectSuccess("unable able to ping %s", hostIP)
+
+			// Flush global conntrack table to be safe because egress conntrack cleanup
+			// is still to be completed (GH-3393).
+			By(fmt.Sprintf("Flushing global connection tracking table before importing policy"))
+			vm.FlushGlobalConntrackTable().ExpectSuccess("Unable to flush global conntrack table")
+
+			By(fmt.Sprintf("Importing policy which allows egress to %s entity from %s", api.EntityHost, helpers.App1))
+			policy := fmt.Sprintf(`
+			[{
+				"endpointSelector": {"matchLabels":{"%s":""}},
+				"egress": [{
+					"toEntities": [
+						"%s"
+					]
+				}]
+			}]`, app1Label, api.EntityHost)
+
+			_, err := vm.PolicyRenderAndImport(policy)
+			Expect(err).To(BeNil(), "Unable to import policy: %s", err)
+
+			areEndpointsReady := vm.WaitEndpointsReady()
```

Comment:
PolicyRenderAndImport already wait until the endpoints are ready, you can delete this assert message ;-)